### PR TITLE
Add images designed by Abbie Brooks.

### PR DESCRIPTION
**Pull request by ntoll**
This branch introduces two changes:

* Inclusion of the designs for additional images created by 11 year old Abbie Brooks.
* Recognition of Abbie's efforts in the `authors` list in the `this` module.

### CODE REVIEW

File: inc/genhdr/qstrdefs.generated.h [hunk 1]
```diff
@@ -382,6 +382,19 @@ QDEF(MP_QSTR_PITCHFORK, (const byte*)"\x33\x09" "PITCHFORK")
...
+QDEF(MP_QSTR_DUCK, (const byte*)"\x3c\x04" "DUCK")
+QDEF(MP_QSTR_HOUSE, (const byte*)"\xa1\x05" "HOUSE")
+QDEF(MP_QSTR_TORTOISE, (const byte*)"\xc8\x08" "TORTOISE")
+QDEF(MP_QSTR_BUTTERFLY, (const byte*)"\x56\x09" "BUTTERFLY")
+QDEF(MP_QSTR_STICKMAN, (const byte*)"\x01\x08" "STICKMAN")
+QDEF(MP_QSTR_GHOST, (const byte*)"\xc2\x05" "GHOST")
+QDEF(MP_QSTR_SWORD, (const byte*)"\x98\x05" "SWORD")
+QDEF(MP_QSTR_GIRAFFE, (const byte*)"\x7d\x07" "GIRAFFE")
+QDEF(MP_QSTR_SKULL, (const byte*)"\x48\x05" "SKULL")
+QDEF(MP_QSTR_BROLLY, (const byte*)"\x63\x06" "BROLLY")
```
Comment (larryhastings) [inline inc/genhdr/qstrdefs.generated.h]: Maybe avoid the slang here and just use "umbrella"?

File: inc/genhdr/qstrdefs.generated.h [hunk 2]
```diff
@@ -382,6 +382,19 @@ QDEF(MP_QSTR_PITCHFORK, (const byte*)"\x33\x09" "PITCHFORK")
 QDEF(MP_QSTR_XMAS, (const byte*)"\xe2\x04" "XMAS")
 QDEF(MP_QSTR_PACMAN, (const byte*)"\x55\x06" "PACMAN")
```
Comment (larryhastings) [inline inc/genhdr/qstrdefs.generated.h]: Maybe I'm being a big scaredycat, but I'd avoid names trademarked by Namco.  How about WEDGE?

p.s. I don't see microbit_const_image_pacman_obj in the diff below, am I missing something?

Comment (ntoll) [inline inc/genhdr/qstrdefs.generated.h]: Hey @larryhastings it's already been merged in some weeks ago. Surely this applies as "fair use" (educational)..? IANAL. ;-)

File: inc/genhdr/qstrdefs.generated.h (hunk 1 above)
Comment (ntoll) [inline inc/genhdr/qstrdefs.generated.h]: I did wonder. But I did take two attempts (speed typing) to type UMBRELLA so went for "brolly". :-) Actually Abbie calls it an umbrella so I'll stick with that. Change incoming.

**Comment by ntoll** [comment]
For the record, this work was donated via emails exchanged with Abbie's mum Sarah Brooks. This is, obviously, an indication of parental consent for Abbie's work to be included in the micro:bit version of MicroPython. :-)

**Comment by dpgeorge** [comment]
Very good!  I'm happy to merge (rebase needed?), but maybe "stickman" should be something gender neutral like "stick_figure" or "stick_person"?

**Comment by ntoll** [comment]
I'll do that now. Give me 10 mins.

**Comment by ntoll** [comment]
Please merge! :-)

**Pull request merged**